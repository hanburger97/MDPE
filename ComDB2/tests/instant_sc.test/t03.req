select i, j, k from t
select i, j, k from t order by i

insert into t(i, j) values(10,10)
insert into t(i, j) values(11,11)
insert into t(i, j) values(12,12)
insert into t(i, j) values(13,13)
insert into t(i, j) values(14,14)
insert into t(i, j, k) values(15, 15, 15)
insert into t(i, j, k) values(16, 16, 16)
insert into t(i, j, k) values(17, 17, 17)
insert into t(i, j, k) values(18, 18, 18)
insert into t(i, j, k) values(19, 19, 19)

select i, j, k from t
select i, j, k from t order by i
