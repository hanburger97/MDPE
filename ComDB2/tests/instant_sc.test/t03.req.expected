(i=0, j=0, k=99)
(i=1, j=1, k=99)
(i=2, j=2, k=99)
(i=3, j=3, k=99)
(i=4, j=4, k=99)
(i=5, j=5, k=99)
(i=6, j=6, k=99)
(i=7, j=7, k=99)
(i=8, j=8, k=99)
(i=9, j=9, k=99)
(i=0, j=0, k=99)
(i=1, j=1, k=99)
(i=2, j=2, k=99)
(i=3, j=3, k=99)
(i=4, j=4, k=99)
(i=5, j=5, k=99)
(i=6, j=6, k=99)
(i=7, j=7, k=99)
(i=8, j=8, k=99)
(i=9, j=9, k=99)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(i=0, j=0, k=99)
(i=1, j=1, k=99)
(i=2, j=2, k=99)
(i=3, j=3, k=99)
(i=4, j=4, k=99)
(i=5, j=5, k=99)
(i=6, j=6, k=99)
(i=7, j=7, k=99)
(i=8, j=8, k=99)
(i=9, j=9, k=99)
(i=10, j=10, k=99)
(i=11, j=11, k=99)
(i=12, j=12, k=99)
(i=13, j=13, k=99)
(i=14, j=14, k=99)
(i=15, j=15, k=15)
(i=16, j=16, k=16)
(i=17, j=17, k=17)
(i=18, j=18, k=18)
(i=19, j=19, k=19)
(i=0, j=0, k=99)
(i=1, j=1, k=99)
(i=2, j=2, k=99)
(i=3, j=3, k=99)
(i=4, j=4, k=99)
(i=5, j=5, k=99)
(i=6, j=6, k=99)
(i=7, j=7, k=99)
(i=8, j=8, k=99)
(i=9, j=9, k=99)
(i=10, j=10, k=99)
(i=11, j=11, k=99)
(i=12, j=12, k=99)
(i=13, j=13, k=99)
(i=14, j=14, k=99)
(i=15, j=15, k=15)
(i=16, j=16, k=16)
(i=17, j=17, k=17)
(i=18, j=18, k=18)
(i=19, j=19, k=19)
